apps/frontend/src/app/Data/Weapons/Bow/TheFirstGreatMagic/index.tsx

```diff
@@ -29,6 +33,11 @@ const moveSPD_arrs = [
 ]
 
 const sameElementTeammates = lookup(input.charEle, tally, naught)
+const otherElementTeammates = sum(
+  ...allElementKeys.map((ele) =>
+    greaterEq(tally[ele], 1, unequal(ele, input.charEle, tally[ele]))
```

Comment:
Why is there greatereq here?

Reply:
I don't know, actually. Copied from Gilded Dreams and it seems to do fine.